### PR TITLE
React native line v2 release

### PR DESCRIPTION
**Pull request by eb16**

### CODE REVIEW

**Review by nicoache1 — APPROVED** [review]

File: _posts/2020-10-02-react-native-line.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+---
+
+# React Native Line Release
+
+# Introduction
+
+We are happy to announce a new [react-native-line](https://github.com/xmartlabs/react-native-line) version that supports the latest updates on the line SDK and it's available to download via [npm](https://www.npmjs.com/package/@xmartlabs/react-native-line)!
+
+
+In the beginning, the motivation to make this library was given that one of our clients had a user base in Japan, and as you may know, Line has a [large number of users](https://www.statista.com/statistics/735063/japan-number-of-line-users/#:~:text=Number%20of%20LINE%20users%20in%20Japan%202016%2D2021&text=In%202016%2C%20approximately%2045.5%20million,58.5%20million%20users%20in%202021.) in that country so it was decided to provide the possibility of logging in with Line in the project.
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:23]:
```suggestion
When one of our projects called  for an integration of sign in with [Line](https://line.me/en/) (which as you may know [lit's pretty popular in Japan](https://www.statista.com/statistics/735063/japan-number-of-line-users/#:~:text=Number%20of%20LINE%20users%20in%20Japan%202016%2D2021&text=In%202016%2C%20approximately%2045.5%20million,58.5%20million%20users%20in%202021.)
```

File: _posts/2020-10-02-react-native-line.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+At that time there were no maintained alternatives in the React Native community to do that, so we decided to develop one.
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:25]:
```suggestion
At the time there were no maintained alternatives in the React Native community to achieve that, so we decided to develop our own!
```

File: _posts/2020-10-02-react-native-line.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+A few years after the first release, being more specific in 2019, an [issue](https://github.com/xmartlabs/react-native-line/issues/31) was reported asking us for support to versions of React Native greater than 0.58.
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:27]:
```suggestion
A few years after the first release, being more specific in 2019, an [issue](https://github.com/xmartlabs/react-native-line/issues/31) was reported asking us for supporting versions of React Native greater than 0.58.
```

File: _posts/2020-10-02-react-native-line.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+The newest React Native version introduced some breaking changes so we spent some time making it compatible again with the latest libraries.
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:28]: This paragraph is not clear to me. In particular the "so we spent some time making it compatible again with the latest libraries"

We had to do a lot of work developing for newer things in RN or to fix compatibility issues with other libraries?

File: _posts/2020-10-02-react-native-line.md [hunk 5]
```diff
@@ -0,0 +1,159 @@
...
+
+Let's get into its functionality and how how it works....
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:30]: This refers to the app as a whole or to the newly implemented changes?

File: _posts/2020-10-02-react-native-line.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+and `botPrompt`:
+
+```jsx
+{
+  aggressive = 'aggressive',
+  normal = 'normal',
+}
+```
+
+This means your application would have the ability to get specific logged user information such as email address, user id, URL, and more!
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:71]:
```suggestion
This means your application will have the ability to get specific information of the logged user, such as its email address, user_id, URL, and more!
```

File: _posts/2020-10-02-react-native-line.md [hunk 7]
```diff
@@ -0,0 +1,159 @@
...
+
+### Logout
+
+It's pretty reasonable that if you log into an application you would like to logout someday. Same as the previous version, this library includes logout functionality.
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:75]:
```suggestion
It's pretty reasonable that if you log into an application, someday you would want to log out. Same as the previous version, this library includes logout functionality.
```

File: _posts/2020-10-02-react-native-line.md [hunk 8]
```diff
@@ -0,0 +1,159 @@
...
+   await Line.logout()
+   ...
+} catch (error) {
+   ...
+}
+```
+
+### Get logged user information
+
+If you want to get information from the logged user, just run this piece of code
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:91]:
```suggestion
If you want to get information from the logged user, just run this snippet of code:
```

File: _posts/2020-10-02-react-native-line.md [hunk 9]
```diff
@@ -0,0 +1,159 @@
...
+### Get bot prompt
+
+Finally, you can get your user friendship status with the Line Official Account. You need to configure it in your account, specify the `botPrompt` option when logging in and then call `getBotFriendshipStatus()` method.
+Follow [LINE's instructions for more information](https://developers.line.biz/en/docs/line-login/link-a-bot/) on how to link the bot to your account.
+
+# Comments about the library
+
+### Implementation
+
+This native module, unlike how usually are made the modules in the React Native community, it's implemented using Swift and Kotlin at a low level.
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:139]:
```suggestion
This native module (unlike most modules made in the React Native community), it's implemented using Swift and Kotlin at a low level.
```
The "Unlike  the RN Community" sounds kinda pretentious to me. I'd consider more changes to the text or get rid of it

File: _posts/2020-10-02-react-native-line.md [hunk 10]
```diff
@@ -0,0 +1,159 @@
...
+
+One of the strongest reasons for choosing Swift as the iOS language is strongly related to some statements LINE has made over the latest version of LineSDK. It is the last version that would be using Objective-C, so they recommend using `Line SDK for iOS Swift` which will be  continuously updated with new features as you can see in the following [page](https://developers.line.biz/en/docs/ios-sdk/objective-c/overview/):
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:141]:
```suggestion
One of our strongest reasons for choosing Swift as the iOS language is strongly related to some statements LINE has made over the latest version of LineSDK. It is the last version that would be using Objective-C, so they recommend using `Line SDK for iOS Swift` which will be  continuously updated with new features as you can see in the following [page](https://developers.line.biz/en/docs/ios-sdk/objective-c/overview/):
```

File: _posts/2020-10-02-react-native-line.md [hunk 11]
```diff
@@ -0,0 +1,159 @@
...
+
+<img src="/images/react-native-line/migrate.png" alt="deprecation advertisement" >
+
+Furthermore, Swift and Kotlin are modern languages that are more intuitive than Objective-C and Java. So we considered the opportunity to challenge us in the use of these two amazing technologies.
+
+# Integration Guide
+
+If you're using LineSDK for the first time, you can check the [installation guide](https://github.com/xmartlabs/react-native-line#installation).
+
+If you want to update to the latest version, please check out our fined-tuned [migration guide](Were you already using the library? Well, just check out our fined-tune migration guide.).
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:151]:
```suggestion
If you want to update to the latest version, please check out our fined-tuned [migration guide](Were you already using the library? Well, just check out our fined-tune migration guide.)
```

File: _posts/2020-10-02-react-native-line.md [hunk 12]
```diff
@@ -0,0 +1,159 @@
...
+
+# Final thoughts
+
+We hope you find it useful. Please contact us if you need any help, find an issue, or struggle with anything.  We are open to any suggestions and to provide help with any issue you might have using the library.
```
Comment (renzodgc) [inline _posts/2020-10-02-react-native-line.md:155]:
```suggestion
We hope you find it useful. Please contact us if you need any help, encounter an issue, or struggle with anything.  We are open to any suggestions and to provide help with any issue you might face using the library.
```

File: _posts/2020-10-02-react-native-line.md [hunk 13]
```diff
@@ -0,0 +1,155 @@
...
+
+<img src="/images/react-native-line/migrate.png" alt="deprecation advertisement" >
+
+Furthermore, Swift and Kotlin are modern languages that are more intuitive than Objective-C and Java. So we considered the opportunity to challenge us in the use of these two amazing technologies.
+
+# Integration Guide
+
+If you're using LineSDK for the first time, you can check the [installation guide](https://github.com/xmartlabs/react-native-line#installation).
+
+If you want to update to the latest version, please check out our fined-tuned [migration guide](Were you already using the library? Well, just check out our fined-tune migration guide.)
```
Comment (matir91) [inline _posts/2020-10-02-react-native-line.md:147]: add the right link to fix the linter issue

**Review by gh-lcardozo — APPROVED** [review]

**Pull request merged**